datafusion/sqllogictest/test_files/string/string.slt

```diff
@@ -34,6 +34,14 @@ statement ok
 create table test_substr as
 select arrow_cast(col1, 'Utf8') as c1 from test_substr_base;
 
+query BBB
+SELECT
+    NULL LIKE '%',
+    '' LIKE '%',
+    'a' LIKE '%'
+----
+NULL true true
```

Comment:
I suggest moving this to `string/string_literal.slt`

Reply:
done!